charts/gateway-addons-helm/values.yaml

```
@@ -42,6 +42,531 @@ prometheus:
     enabled: false
   kube-state-metrics:
     enabled: false
+    customResourceState:
+      enabled: true
+      # Add (Cluster)Role permissions to list/watch the customResources defined in the config to rbac.extraRules
+      config:
+        kind: CustomResourceStateMetrics
+        spec:
+          resources:
+            - groupVersionKind:
+                group: gateway.networking.k8s.io
+                kind: "Gateway"
+                version: "v1beta1"
```

```suggestion
                version: "v1"
```
shouldn't this be `v1`?

---

updated api versions